CRM/CivirulesActions/Participant/AddToZoom.php

```diff
@@ -130,6 +130,9 @@ private function addParticipant($participant, $entityID, $triggerData, $entity)
 			'Content-Type' => 'application/json;charset=UTF-8',
 			'Authorization' => "Bearer $token"
 		])->post($url, $participant);
+		CRM_Core_Error::debug_var('Zoom addParticipant event Id', $event['id']);
```

Comment:
@damo-veda I would expect you to use civirules logged (see below code for example) to log the data/values, instead of CRM_Core_Error::debug_var, as users cannot see the data/values logged using debug_var. Any data added using civirules logged can be viewed in UI.

`$this->logAction($msg, $triggerData, \PSR\Log\LogLevel::INFO);`